src/resolvers/event.js

```diff
@@ -83,10 +83,10 @@ module.exports = {
      * @param {number} timezoneOffset - user's local timezone offset in minutes
      * @returns {Promise<ProjectChartItem[]>}
      */
-    async chartData({ projectId, groupHash }, { days, timezoneOffset }, context) {
+    async chartData({ projectId, groupHash }, { startDate, endDate, groupBy, timezoneOffset }, context) {
       const factory = getEventsFactory(context, projectId);
 
-      return factory.findChartData(days, timezoneOffset, groupHash);
+      return factory.getChartData(startDate, endDate, groupBy, timezoneOffset, projectId, groupHash);
```

Comment:
"Days Repeating" chart in Event Overview does not work because we don't have such data in redis

<img width="960" height="640" alt="Image" src="https://github.com/user-attachments/assets/c2aa783d-4832-4f90-afc1-657a3615951b" />